kubernetes/kubespray/includes/application.yaml

```diff
@@ -115,6 +119,9 @@ node_templates:
         configure:
           implementation: ansible.cloudify_ansible.tasks.run
           inputs:
+            extra_packages:
+            - netaddr
+            - ipaddr
```

Comment:
What about the yum packages?

Reply:
Tested on 505 and 5.1 there is a script that installs python-netaddr and it works.